bin/restore-database-backups.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+#
+# Run this script to loop over .sql files inside ./database-backups
+# and restore their databases to the running mysql-business-analytics container
+#
+
+function restoreDatabaseBackup() {
+   echo "Processing $1"
+   cat "$1" | docker exec -i ba-mysql /usr/bin/mysql --user=root --password=root
```

Comment:
This is only for the database running locally on the dev's machine, and to be honest this is an issue throughout, and not one I think we're going to be tackling any time soon. Since those databases aren't exposed to the internet, we've considered that "good enough".

Reply:
Yeah I think knowing the credentials only means that if you were to clone our repo, run the containers, run this script to restore some mysql backups, then you could access that mysql container. But the mysql container would only container database backups that you had access to anyway.

Reply:
Right. Brec's point makes sense to me. I think in the longer run we're going to want safer data handling procedures, but the credentials here aren't the right fix.